<a href="https://colab.research.google.com/github/j-claudinei-f/j-claudinei-f/blob/main/C%C3%B3pia_de_Av_2_Mat_Discreta.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#**Avaliação II: Possível gabarito**

José Claudinei Ferreira

Universidade Federal de Alfenas

Seguem problemas para resolverem como parte da avaliação.

---

1. (1 ponto) Se $a_1=1$ e $$a_{n+1}=2a_n+n+1.$$ Determine $a_n$, em função apenas de $n$.

Dica: Veja exemplo neste [link](https://github.com/j-claudinei-f/j-claudinei-f/blob/main/Recorr%C3%AAncias_ordem1.ipynb).

**Resolução:**

Se olharmos para a relação de recorência dada e considerarmos a parte $b_{n+1}=2b_n$, encontramos a solução da forma $b_n=c2^n$. Podemos ecolher $c=1$.

Agora, seja $y_n$ tal que $2^ny_n=a_n$. Segue, da relação de recorrência dada, que $$\begin{cases}2^{n+1}y_{n+1}&=&a_{n+1}\\\\&=&2a_n+n+1\\\\&=&2\times 2^ny_n+n+1\end{cases},$$ ou
que $$\begin{cases}y_{n+1}&=&y_n+\frac{n+1}{2^{n+1}}\\\\y_1=\frac{1}{2}\end{cases}.$$

Sendo assim, vamos encontrar $y_n$ e depois usá-lo para encontrar $a_n$.

Como $$\begin{cases}y_1&=&\frac{1}{2}\\\\
y_2&=&y_1+\frac{2}{2^2}=\frac{1}{2}+\frac{2}{2^2}\\
\\
y_3&=&y_2+\frac{3}{2^3}=\frac{1}{2}+\frac{2}{2^2}+\frac{3}{2^3}\\\vdots&\vdots&\vdots\\y_n&=&y_{n-1}+\frac{n}{2^n}=\frac{1}{2}+\frac{2}{2^2}+\cdots+\frac{n}{2^n}\\\\&=&\sum_{j=1}^n\frac{j}{2^j}\end{cases}$$

Portanto, $$a_n=2^ny_n=2^n\sum_{j=1}^n\frac{j}{2^j},$$ e podemos melhorar essa expressão pensando no somatório
$$\sum_{j=1}^n\frac{j}{2^j}.\tag{1}$$



Como essa soma não é muito conhecida, podemos olhar para a dica dada, que remete a pensar com o operador diferenças $$\Delta f_n=f_{n+1}-f_n$$ e numa variação envolvendo produto de sequências, dado por
$$\begin{cases}\Delta (f_ng_n)&=&f_{n+1}g_{n+1}-f_ng_n\\\\&=&f_{n+1}g_{n+1}-f_{n+1}g_n+f_{n+1}g_n-f_ng_n\\\\&=&f_{n+1}\Delta g_n+g_n\Delta f_n\end{cases}$$ Essa última expressão nos leva à igualdade
$$\Delta (f_ng_n)-g_n\Delta f_n=f_{n+1}\Delta g_n.\tag{2}$$

Observando agora que, se tomarmos $f_n=n$ e $g_n=2^{-n}$, vamos obter
$$\Delta g_j=\frac{1}{2^{j+1}}-\frac{1}{2^j}=-\frac{1}{2^{j+1}}.$$
Com isso, podemos escrever, cada termo do somatório (1) como
$$\frac{j}{2^j}=-f_j\Delta g_{j-1}, $$ ou
$$\frac{j+1}{2^{j+1}}=-f_{j+1}\Delta g_{j}=-\left(\Delta f_jg_j-g_j\Delta f_j\right)=-\Delta f_jg_j+g_j\Delta f_j,$$ tendo em mente a Expressão (2).

Reescrevendo o somatório (1), levando isso em consideração, obtemos
$$\sum_{j=1}^n\frac{j}{2^j}=\sum_{j=0}^{n-1}\frac{j+1}{2^{j+1}}=-\sum_{j=0}^{n-1}f_{j+1}\Delta g_j=-\color{blue}{\sum_{j=0}^{n-1}\Delta (f_{j} g_j})+\color{green}{\sum_{j=0}^{n-1}g_{j}\Delta f_j}.\tag{$1^*$}$$

Fazendo as substituições das últimas duas somas, obtemos
$$\sum_{j=1}^n\frac{j}{2^j}=-\color{blue}{\sum_{j=0}^{n-1}\Delta (f_{j} g_j)}+\color{green}{\sum_{j=0}^{n-1}g_{j}\Delta f_j}=-\color{blue}{(f_ng_n-f_0g_0)}+\color{green}{\sum_{j=0}^{n-1}g_{j}},\tag{$1^{**}$}$$ porque $\Delta f_n=1$ e $$\sum_{j=0}^{n-1}\Delta (f_{j} g_j)=f_ng_n-f_0g_0.$$

Finalmente,
$$a_n=2^n\sum_{j=1}^n\frac{j}{2^j}=2^n\left(-\color{blue}{\left(\frac{n}{2^n}-\frac{0}{2^0}\right)}+\color{green}{\frac{1-\left(\frac{1}{2}\right)^n}{1-\frac{1}{2}}}\right)=-\color{blue}{n}+\color{green}{2^{n+1}-2},\tag{3}$$ lembrando que $$\color{green}{\sum_{j=0}^{n-1}g_{j}}=1+\frac{1}{2}+\cdots+\left(\frac{1}{2}\right)^{n-1}=\frac{1-\left(\frac{1}{2}\right)^n}{1-\frac{1}{2}}.$$



Você pode checar seus cálculos usando a relação de recorrência para $a_n$ dada no ínício do problema.

In [ ]:
a=[0]                  # a(1)=a(0+1)=2*a(0)+0+1 e a(0)=0
for i in range(0,10):
  a.append(2*a[i]+i+1)      # a(n+1)=2a(n)+n+1
a

[0, 1, 4, 11, 26, 57, 120, 247, 502, 1013, 2036]

Para a expressão obtida na Expressão (3):

In [ ]:
def a(n):                # função a(n), para cada n natural
  p=-n+2**(n+1)-2
  return p

q=[]
for i in range(0,10+1):
  q.append(a(i))

q

[0, 1, 4, 11, 26, 57, 120, 247, 502, 1013, 2036]

---

2. (1 ponto) Se $a_1=a_2=1$ e $$a_{n+1}=3a_n-2a_{n-1}+n+1.$$ Determine $a_n$, em função apenas de $n$.



**Resolução:**

Uma forma de resolver esse problema é escrever a relação de recorrência dada como $$\begin{cases}a_{n+1}-a_n&=&2(a_n-a_{n-1})+n+1\\a_1&=&1\\a_2&=&1\end{cases}.$$

Escrevendo $b_n=\Delta a_n=a_{n+1}-a_n$, obtemos uma nova relação de recorrência
$$\begin{cases}b_n&=&2b_{n-1}+n+1\\b_1&=&0\end{cases},$$ ou
$$\begin{cases}b_{n+1}&=&2b_{n}+n+2\\b_1&=&0\end{cases},$$ que podemos reaolver pelo método da Questão 1.




Agora, seja $y_n$ tal que $2^ny_n=b_n$. Segue, da relação de recorrência logo acima, que
que $$\begin{cases}y_{n+1}&=&y_n+\frac{n+2}{2^{n+1}}\\\\y_1=0\end{cases}.$$

Sendo assim, vamos encontrar $y_n$ e depois usá-lo para encontrar $b_n$.

Como $$\begin{cases}y_1&=&0\\\\
y_2&=&y_1+\frac{3}{2^2}=\frac{3}{2^2}\\
\\
y_3&=&y_2+\frac{4}{2^3}=\frac{3}{2^2}+\frac{4}{2^3}\\\vdots&\vdots&\vdots\\y_n&=&y_{n-1}+\frac{n+1}{2^n}=\frac{3}{2^2}+\frac{4}{2^3}+\cdots+\frac{n+1}{2^n}\\\\&=&\sum_{j=2}^n\frac{j+1}{2^j}\\\\&=&\sum_{j=2}^n\frac{j}{2^j}+\sum_{j=2}^n\frac{1}{2^j}\end{cases}$$

Portanto, $$b_n=2^ny_n=2^n\left(\sum_{j=2}^n\frac{j}{2^j}+\sum_{j=2}^n\frac{1}{2^j}\right).$$

Já sabemos, também da Questão 1, que
$$2^n\sum_{j=1}^n\frac{j}{2^j}=-{n}+{2^{n+1}-2},$$ e que sabemos que $$\sum_{j=0}^n\frac{1}{2^j}=1+\frac{1}{2}+\cdots+\left(\frac{1}{2}\right)^{n}=\frac{1-\left(\frac{1}{2}\right)^{n+1}}{1-\frac{1}{2}}=2\left(1-\left(\frac{1}{2}\right)^{n+1}\right)=2-\frac{1}{2^{n}}.$$




Então $$b_n=\left(\left(2^{n+1}-(n+2)\right)-2^n\times \frac{1}{2}\right)+2^n\left(2-\left(\frac{1}{2^{n}}\right)-1-\frac{1}{2}\right)$$ ou
$$b_n=2^{n+1}-(n+3).$$

In [ ]:
b=[-1]                  # b(1)=b(0+1)=2*b(0)+0+2 e b(0)=-1
for i in range(0,10):
  b.append(2*b[i]+i+2)      # b(n+1)=2b(n)+n+2
b

[-1, 0, 3, 10, 25, 56, 119, 246, 501, 1012, 2035]

Com a forma encontrada:

In [ ]:
def b(n):                # função a(n), para cada n natural
  p=-n+2**(n+1)-3
  return p

q=[]
for i in range(0,10+1):
  q.append(b(i))

q

[-1, 0, 3, 10, 25, 56, 119, 246, 501, 1012, 2035]

Ainda precisamos encontrar $a_n$ tal que $$b_n=a_{n+1}-a_n=2^{n+1}-(n+3),$$ ou
$$\begin{cases}a_{n+1}&=&a_n+2^{n+1}-(n+3)\\a_1&=&1\end{cases}.$$

Isso nos diz que
 $$\begin{cases}a_n&=&a_{n-1}+2^n-(n+2)\\\\
 &=&a_{n-2}+2^{n-1}+2^{n}-((n+1)+(n+2))\\\\
 &=&a_2+2^3+2^4+\cdots +2^n-(5+\cdots+(n+1)+(n+2))\\\\&=&
 1+2^{3}(1+2+\cdots+2^{n-3})-\frac{(5+(n+2))(n-2)}{2}\\\\&=&1+2^3(2^{n-2}-1)-\frac{(n-2)(n+7)}{2}\\\\&=&2^{n+1}-7-\frac{(n-2)(n+7)}{2}\\\\&=&2^{n+1}-\frac{n(n+5)}{2}\end{cases}$$

Testanto os resultados pela relação de recorrência:

In [ ]:
a=[2,1]                  # a(1)=a(2)=1
for i in range(1,10):
  a.append(3*a[i]-2*a[i-1]+i+1)      # a(n+1)=3a(n)-2a(n-1)+n+1
a

[2, 1, 1, 4, 14, 39, 95, 214, 460, 961, 1973]

Ou

In [ ]:
a=[2]                  # a(2)=1=a(1) e a(0)=2
for i in range(0,10):
  a.append(a[i]+2**(i+1)-(i+3))      # a(n+1)=a(n)+2^(n+1)-(n+3)
a

[2, 1, 1, 4, 14, 39, 95, 214, 460, 961, 1973]

Pela exprassão em função de $n$:

In [ ]:
def a(n):                # função a(n), para cada n natural
  p=2**(n+1)-7-((n+7)*(n-2))/2
  return p

q=[]
for i in range(0,10+1):
  q.append(a(i))

q

[2.0, 1.0, 1.0, 4.0, 14.0, 39.0, 95.0, 214.0, 460.0, 961.0, 1973.0]

**Obs:** Essa questão poderia ser resolvida primeiro encontrando as soluções da equação $$b_{n+1}-3b_n+2b_{n-1}=0$$ e depois procurando soluções particulares da equação
$$b_{n+1}-3b_n+2b_{n-1}=n+1,$$ tais que $a_1=a_2=1$, por tentativa, como é feito no livro texto.

Em todo, caso é preciso pensar se a tentativa gera solução correta, porque pode haver erro de cálculo ou de chute de forma da resposta.

---


3. (2 pontos) Um apartamamento na planta, de 35.7 metros quadrados, na Praia das Toninhas, no Guarujá, é colocado a venda nas seguintes condições:

A entrada é parcelada em 12 vezes de $R\$ 10.351,04$, com um mês de carência. O restante deverá ser pago em 42 prestações de $R\$ 2.261,00$. Além disso terão 4 prestações intermediárias de $R\$ 10.175,00$, de seis em seis meses.

a) Sabendo que a taxa de juros do financiamento é de $12\%$ ao ano. Determine o valor real do apartamento, em caso de pagamanto a vista.

b) Se o saldo devedor for corrigido mensalmente pela tabela TR, qual seria o saldo restante após o final dos pagamentos. Aqui, considere a TR média de 2022, porque não tem como prever a TR futura.

**Resolução:**

a) Como a taxa de juros $j$ do processo de financiamento é de $12\%$ ao ano, vamos transformá-la em taxa mensal primeiro. Para isso, seja $i$ a taxa de juros mensal e $C$ um capital investido. Então $$C(1+12/100)=C(1+i)^{12}.$$

Isso nos dá $$i=\sqrt[12]{1+12/100}-1\approx 0.009488.$$

In [20]:
i=(1+12/100)**(1/12)-1
i

0.009488792934583046

Segue que o valor à vista do pequeno apartamento pode ser calculando trazendo para o presente os valores que serão pagos no futuroe depois somando-os.

Aqui nesse editor de texto, parace-me melhor trazer os três tipos de pagamendo separadamente e depois somá-los.

Seja $E=10.351,04$ o valor das 12 parcelas da entrada, com um mês de carência. Seja $P=2.261,00$ o valor das 42 prestações. Por fim, seja $I=10.175,00$ o valor das 4 prestações semestrais.


Usando o índice $p$ para o valor presente, devemos ter
$$\begin{cases}E_p&=&\frac{E}{1+i}+\frac{E}{(1+i)^2}+\cdots +\frac{E}{(1+i)^{12}}\\\\P_p&=&\frac{P}{1+i}+\frac{P}{(1+i)^2}+\cdots +\frac{P}{(1+i)^{42}}\\\\I_p&=&\frac{I}{(1+i)^6}+\frac{I}{(1+i)^{12}}+\frac{I}{(1+i)^{18}}+\frac{I}{(1+i)^{24}}\end{cases}$$

In [27]:
E=10351.04
P=2261.00
I=10175.00

Ep=0
for k in range(1,12+1):
  Ep=Ep+E/((1+i)**k)
Pp=0
for k in range(1,42+1):
  Pp=Pp+P/((1+i)**k)
Ip=0
for k in range(1,4+1):
  Ip=Ip+I/((1+i)**(6*k))

Vp=Ep+Pp+Ip
Vp


230294.61411247842

**Obs:** Poderia ser entendido do enunciado que a prestação é com um mês de carência ou não. É diferente mas aceitaria os dois entendimentos. Eu considerei a [carência](https://pt.wikipedia.org/wiki/Per%C3%ADodo_de_car%C3%AAncia) na resolução apresentada, ou seja, começa-se a pagar as prestações e entrada no início do segundo mês.

b) Neste item eu usei uma tabela da TR de 2022 que já estava em arquivo que disponibilizei antes.

In [9]:
TR=[0.06,0,0.1,0.06,0.17,0.15,0.16,0.24,0.18,0.15,0.15,0.21]     # Taxa referencial de janeiro a dezembro de 2022; Em janeiro foi de 0.06%, por exemplo.
import pandas as pd    # Para lidar com tabelas

df = pd.DataFrame(TR,columns=['Taxa TR'],index=['janeiro','fevereiro','março','abril','maio','junho','julho','agosto','setembro','outubro','novembro','dezembro'])
print(df)

           Taxa TR
janeiro       0.06
fevereiro     0.00
março         0.10
abril         0.06
maio          0.17
junho         0.15
julho         0.16
agosto        0.24
setembro      0.18
outubro       0.15
novembro      0.15
dezembro      0.21


No caso, a TR acumulada (ou TR nominal) anual foi de

In [22]:
import numpy as np
TR=np.array(TR)
J=np.sum(TR)/100
J

0.016299999999999995

ou $1.63\%$ ao ano.

Cabe notar que a taxa nominal (ou acumulada) é a taxa de juros  simples usada para correção de um capital (ou saldo devedor) $C$ em um ano. Mas estamos interessados na taxa de juros compostos.

A taxa media de juros (mensais) compostos é dada pela expressão $$C\prod_{k=1}^{12}(1+t_k)=C(1+t)^{12},$$ em que $t_k$ é a taxa TR no mês $k$.

Isso nos diz que $$t=\sqrt[12]{\prod_{k=1}^{12}(1+t_k)}-1\approx 0.0013581173631236343 $$

In [23]:
p=1
for k in range(0,12):
  p=p*(1+TR[k]/100)
t=p**(1/12)-1
t

0.0013581173631236343

Ou $0.1358\%$ ao mês.

Se o saldo devedor for corrigido mensalmente pela TR média e não for distribuido nas prestações, haverá saldo devedor ao final do pagamento das parcelas. Para calcular este saldo basta corrigirmos o saldo devedor em $i+t$ mais a TR mensalmente.

In [34]:
j=i+t
j

0.01084691029770668

Agora será necessário usar os valores $P_k$ pagos em cada mês $k$:

In [36]:
Prestacoes=[0]
for k in range(1,42+1): # 42 prestações de valor 2261.0
  Prestacoes.append(P)

Prestacoes[6]=Prestacoes[6]+I  # 4 prestações intermediárias
Prestacoes[12]=Prestacoes[12]+I
Prestacoes[18]=Prestacoes[18]+I
Prestacoes[24]=Prestacoes[24]+I

for k in range(1,12+1): # 12 prestações da entrada
  Prestacoes[k]=Prestacoes[k]+E

Prestacoes

[0,
 12612.04,
 12612.04,
 12612.04,
 12612.04,
 12612.04,
 22787.04,
 12612.04,
 12612.04,
 12612.04,
 12612.04,
 12612.04,
 22787.04,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 12436.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 12436.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0,
 2261.0]

O saldo devedor $S_k$ do mês $k$ será o saldo $S_{k-1}$ do mês anterior, acrescido de juros a taxa $i+t$ menos a prestação $P_k$ paga no mês $k$, ou seja,
$$S_k=S_{k-1}(1+i+t)-P_k,\qquad S_0=V_p.$$

In [38]:
Saldo=[Vp]
for k in range(1,42+1): # Correção de saldo devedor
  Saldo.append(Saldo[k-1]*(1+i+t)-Prestacoes[k])

Saldo

[230294.61411247842,
 220180.55913380143,
 209956.79790802466,
 199622.14046132672,
 189175.38391234694,
 178615.31233217838,
 157765.69660284236,
 146864.9269619486,
 135845.9176505841,
 124707.38613364962,
 113448.03596450278,
 102066.55663406075,
 80386.62341826622,
 78997.56991161968,
 77593.44946618783,
 76174.0986522372,
 74739.35226732667,
 73289.04331707906,
 61648.00299574416,
 60055.693354271745,
 58446.11207295211,
 56819.07180785714,
 55174.38318295592,
 53511.85476807274,
 41656.29305660594,
 39847.135130725925,
 38018.353431109506,
 36169.73510044326,
 34301.06497256958,
 32412.125547442847,
 30502.696965813968,
 28572.556983640283,
 26621.48094621794,
 24649.241762033675,
 22655.60987633294,
 20640.35324440136,
 18603.23730455636,
 16544.024950845836,
 14462.476505450682,
 12358.349690787996,
 10231.399601311665,
 8081.378675007085,
 5908.036664576686]

E a correção com a taxa média da TR de 2022 gera saldo devedor de $R\$5908.04$ após o pagamento das 42 prestações.

---

4. (2 pontos ) Resolva o problema do Exemplo 9 deste [link](https://github.com/j-claudinei-f/j-claudinei-f/blob/main/Combinatoria.ipynb).


O problema em questão é:
Um casal de amigos meus gosta de duas coisas, a primeira é ter filhos e a segunda é colocar o nome do próximo filho, usando alguma regra.

Uma filha deles chama-se Amarilis e eles resolveram que o nome da próxima criança terá apenas seis letras, que essas letras precisam estar no nome de Amarilis, sendo permitido haver até dois a's ou até dois i's.

Eles acham que Silmar é uma possibilidade, se a criança for menina, por exemplo.

Eles sabem que alguns agrupamentos de letras não serão nomes de pessoas, digamos aceitos, mas eles querem pensar sobre a quantidade de expressões (ordenações) de seis letras do nome Amarilis.

Quantos são esses agrupamentos?

**Resolução:**
A palavra Amarilis tem 8 letras, sendo que o A repete duas vezes e I repente duas vezes também.

Podemos contar o tanto de expressões distintas e formadas por sei letras dessa palavra (podendo ter dois A ou dois I) dividindo em casos:

1.   Considere o caso em que não são repetidas letras. Temos apenas seis letras para trocarmos de lugar. Isso nos dá $p_1$ possibilidades, em que
$$p_1=6!=720.$$


In [ ]:
import math

p1=math.factorial(6)
p1

720

2.   Agora repetimos o A ou o I.
a) Note que, se repetirmos o A, teremos seis letras, sendo outras 5 letras distintas para escolhermos 4. Ou seja, vai sobrar uma letra em cada caso. Teremos 5 casos para pensar.

Fixada uma letra, por exemplo $R$, que vai ficar fora, teremos 6 letras para trocarmos de lugar, sendo duas delas identicas. É um caso de permutação com repetição.

Segue que $p_2$ é 2 vezes 5 vezes o número de casos, com uma letra fixada, para ficar de fora, porque o caso de repetir I é identico. Assim,
$$p_2=2\times 5\times \times P_6^{2}=10\times\frac{6!}{2!}=10 \times 360=3600.$$

In [ ]:
p2=10*math.factorial(6)/math.factorial(2)
p2

3600.0

3. Agora repetimos o A e o I duas veves. Nesse caso, usaremos quatro das seis letras distintas e duas vão ficar de fora, e duas (diferentes de A e I) entram, de cada vez. Então a quantidade de possibilidades de escolhas das duas letras que entram é a combinação de 4 dois a dois, que é $$\frac{4!}{2!(4-2)!}=6.$$

Assim, $p_3$ é dado por
$$p_3=6\times P_6^{2,2}=6\times \frac{6!}{2!2!}=6\times 180=1080.$$

In [ ]:
p3=math.comb(4,2)*math.factorial(6)/(math.factorial(2)*math.factorial(2))
p3

1080.0

O total de casos será $p_1+p_2+p_3=5400.$

In [ ]:
p1+p2+p3

5400.0

---

5. Um [poliedro](https://pt.wikipedia.org/wiki/Poliedro) é um objeto tridimensional, formado pela união de polígonos. Esses polígonos são chamados de faces do poliedro. Os lados dos polígonos são chamados de arestas do poliedro e os vértices dos polígonos (juntos) são chamados de vértices do poliedro.

a) (1 ponto) Sejam $P_1,\,P_2,\,\ldots P_n$ os vértices (distintos) de um polígono de $n$ lados. Mostre que a soma dos ângulos internos desse polígono é $(n-2)\times 180$ graus.

b) (1 ponto) Seja $F$ o número de faces e $A$ o número de arestas de um poliedro. Suponha que todas as faces desse poliedro são polígonos de $n$ lados. Mostre que vale a relação $nF=2A$.

c) (1 ponto) Seja $F$ o número de faces, $A$ o número de arestas e $V$ o número de vértices de um poliedro $Q$. Você pode incluir algumas diagonais em cada face e obter outras faces e outras arestas, sem aumentar o número de vértices e de tal forma que o poliedro possa ser identificado com um novo poliedro $R$, cujas faces são regiões triangulares.

Seja $\mathcal{F}$ o número de faces, $\mathcal{A}$ o número de arestas e $\mathcal{V}$ o número de vértices do poliedro $R$. Mostre que $$V+A-F=\mathcal{V}+\mathcal{A}-\mathcal{F}.\tag{$\star$}$$

**Resolução:**

a) Pode-se mostrar, usando retas paralelas (ou recortando papel e dobrando) que a soma dos ângulos internos de um triângulo é $180^o$. Então, para $n=3$ lados de uma região poligonal, vale a igualdade $s_n=(n-2)180^o$, em que $s_n$ denota a soma dos ângulos internos de uma região poligonal de $n$ lados.

Supondo, como hipótese de indução finita que $s_j=(j-2)180^o$, para todo $3\leq j\leq n$, é a soma dos ângulos internos de toda região poligonal com $j$ lados.

Considere uma região poligonal $R$ com $n+1$ lados. Escolha uma diagonal dessa região poligonal, que esteja contida em seu interior. Essa diagonal divide a região em duas outras regiões, digamos, $R_1$, com $p$ lados, e $R_2$, com $q$ lados, de tal forma que $p+q=n+3$, com $p\leq n$ e $q\leq n$, porque a diagonal escolhida é contada duas vez, e os demais lados de $R_1$ e $R_2$ estão na região $R$. Segue da hipótese de indução que a soma dos ângulos internos dessas regiões é $s_p=(p-2)180^o$ e $s_q=(q-2)180^o$.

Ainda $s_n=s_p+s_q=(p+q-4)180^o=((n+1)-2)180^o$. Segue que $s_n=(n-2)180^o$ para todo $n$ natural e maior que 2.

b) Considere um poliedro $P$, cujas faces possuem $n$ lados cada uma. Esses lados são chamados de arestas do poliedro.

Seja $F$ o número de faces e $A$ o número de arestas desse poliedro $P$.

Podemos fazer uma cópia de cada face do poliedro em outro lugar. Teremos então $F$ cópias de faces. Depois fazemos uma cópia de cada lado das cópias das faces. Como cada face tem $n$ lados teremos feito $nF$ cópias de lados (que são cópias de arestas).

Podemos também fazer uma cópia de cada uma das arestas de cada face do poliedro $P$. Como cada aresta está em exatamente duas faces faremos duas cópias de cada aresta, obtendo então $2A$ cópias de arestas.

As duas formas de fazermos cópias de arestas deve dar o mesmo número de cópias, como você pode pensar a respeito.

Então $nF=2A$.


c) Seja $F$ o número de faces, $A$ o número de arestas e $V$ o número de vértices de um poliedro $Q$. Você pode incluir algumas diagonais em cada face e obter outras faces e outras arestas, sem aumentar o número de vértices. Isso ocorre porque as diagonais partem de (ou chegam em) um vértices existentes.

Para cada aresta incluída obtemos uma face nova. Então, se $\mathcal{F}$ for o número de faces, $\mathcal{A}$ o número de arestas e $\mathcal{V}$ o número de vértices do poliedro $R$, que obtemos após a inclusão de aresta (diagonais) novas e, consequentemente, de faces novas. Devemos ter $$A-F=\mathcal{A}-\mathcal{F},\qquad V=\mathcal{V},$$ ou
$$V+A-F=\mathcal{V}+\mathcal{A}-\mathcal{F}.\tag{$\star$}$$

Em particular, podemos adicionar diagonais até que as faces fiquem triângulares.

**Obs:** Você não pode provar isso usando a relação de Euler (Exercício 6), porque ela depende desse resultado. Ainda, não foi dito que o poliedro satisfaz à relação de Euler.

---

**Obs:**

*   Entregar resolução manuscrita, com detalhes da argumentação, até 10 horas, do dia 23 de junho, no meio da aula.
* A avaliação vale 10 pontos. Então escolha um dos itens que valem 1 ponto, para valer 2 pontos. Ou resolva o exércício 6.
* Lembre-se que a ideia é que aprendam com a resolução.
* Para quem estiver com tempo sugiro leitura do texto deste [link](https://pt.wikipedia.org/wiki/Caracter%C3%ADstica_de_Euler#).

---

**Extra:**
6. (1 ponto) Seja $F$ o número de faces, $A$ o número de arestas e $V$ o número de vértices de um [poliedro convexo](https://pt.wikipedia.org/wiki/Poliedro#Poliedro_Convexo) $Q$. Mostre, usando o Exercício 5, que $$V+A-F=2.\tag{$\star$}$$

**Dica:** Como o número de arestas é finito. Você pode escolher um vértice e pendurar o poliedro por este vértice, usando uma linha por exemplo. Se fizer isso embaixo de uma lámpada, pode ver uma sombra no chão, ou numa mesa. Essa sombra será uma região poligonal $S$, em que cada lado $l_i$ dessa região é a sombra de alguma aresta do poliedro.

Pode observar que algumas faces do poliedro ficarão iluminadas (recebem luz direta da lâmpada) e outras ficarão na sombra (não recebem luz direta da lâmpada). A separação entre as faces iluminadas e as outras é dada exetamente pelas arestas que geram os lados $l_i$.

Pense agora na soma dos ângulos internos das faces do poliedro. Depois pense em como ficam as somas desses ângulos, na sombra da região iluminada e na sombra da região não iluminada.